community/shell/src/main/java/org/neo4j/shell/StartClient.java

```diff
@@ -140,13 +151,13 @@ private void start( String[] arguments, CtrlCHandler signalHandler )
         String pid = args.get( ARG_PID, null );
 
         if ( (path != null && (port != null || name != null || host != null || pid != null))
-                || (pid != null && host != null) )
+             || (pid != null && host != null) )
         {
             System.err.println( "You have supplied both " +
```

Comment:
I redid the way this class handles `PrintStream`s by adding fields. What do you think?